### DISCUSSION TITLE
Choose a game pop up.

### DISCUSSION THREAD

**Discussion by learnfromit** [MikeMaximus/gbm#255] [Q&A]
I sometimes play unknown games, every time after playing it pops up "choose a game" that you just played. Is there a way to disable this pop up and just have it ignore those games? couldnt find this in any settings. thanks.

![image](https://user-images.githubusercontent.com/47046874/152291822-89246ecb-f72f-4383-9236-1eb9dca9e0b1.png)

**Comment by MikeMaximus** [comment]
Hello there,

This window appears because all those games use the same executable name (Game.exe), this makes it impossible for GBM to identify them the first time they are played.  In normal cases this window should only appear once,  once you select the game you were playing GBM remembers the installation location and uses that to identify the game in the future.

However if the game is running as administrator, GBM won't be able to detect the installation location and the window will appear every time you play the game.  That leaves you with the following options.

1.  Run GBM as administrator before playing the game.  You can do this quickly by using the blue (User Icon) button on the bottom left of the main GBM window to switch.  Note that GBM will still need to record the installation location,  so the Choose Game window will appear one more time.
2.  Edit the configuration of the games in the list you don't play and disable monitoring.  You can do this by editing the game configurations in the Game Manager.  Select the game, click the Monitor Options (Magnifying Glass) button and uncheck "Monitor this game".  
3. You can also just delete the configurations of the games you don't play.

I hope this helps.

**Reply by basxto** [reply]
It sounded more like the game isn’t in the list/isn’t known by GBM.
So all of those listed games would need to be removed or a new button added to put the detected path into a blocklist.

**Reply by MikeMaximus** [reply]
Ah right, Thanks basxto.

If that's the case, use option 2 or 3 from my previous reply for every game in the list to prevent that window from appearing again.
